docs: pgaudit.log_parameter is not supported (#20184)
supabase/supabase: apps/docs/pages/guides/database/extensions/pgaudit.mdx

For a full list of available settings see [settings docs](https://github.com/pgaudit/pgaudit/blob/master/README.md#settings). Be aware that the `all` setting will generate a very large volume of logs.

> <Admonition>

Reviewer: ```suggestion
<Admonition type="note">
```

> We don't currently support configuring `pgaudit.log_parameter` because it may log secrets in encrypted columns if you happen to be using [pgsodium](/docs/guides/database/extensions/pgsodium) or [Vault](/docs/guides/database/vault).

Reviewer: ```suggestion
We don't currently support configuring `pgaudit.log_parameter` because it may log secrets in encrypted columns if you are using [pgsodium](/docs/guides/database/extensions/pgsodium) or [Vault](/docs/guides/database/vault).
```
